internal/provider/generate.go

```diff
@@ -446,7 +450,8 @@ func (g *generator) renderStaticWebsite(providerName string, providerSchema *tfj
 		case "": // provider
 			if relFile == "index.md.tmpl" {
 				tmpl := providerTemplate(tmplData)
-				render, err := tmpl.Render(providerName, g.renderedProviderName, "", providerSchema.ConfigSchema)
+				exampleFilePath := filepath.Join(g.examplesDir, "provider", "provider.tf")
```

Comment:
this is an assumption i've made to follow the remainder of the tooling `examples/<entity_type>/<entity_type>.tf`. happy to change this if we think there is a better convention to go with here.

also, most implementations i've found of this, people have hardcoded so might not be too big of a deal.

Reply:
This is, as far as I know, correct. It's in the doc.

Given you are looking at fixing this bug, I have taken the liberty to add a commit here to better separate and the doc for those key paths. I have essentially separated templates from examples, hopefully to better highlight those paths.

Reply:
great idea, thanks.